docs/application/dotnet/guides/hot-realoading/beholder.md

```diff
@@ -0,0 +1,75 @@
+# Introduce Beholder
+
+Beholder is a hot reloading support tool for programs using Tizen.NET's NUI.
+
+Hot reloading refreshes the files that were changed without losing the state of the app.
+
+After launching the application on the emulator or real device, you can apply the changed file without stopping the application.
+
+If even in the screen that looks only go through several stages, without a step back, you can update the screen is changed files are applied
+
+
+## Hot Reloading 
+
+ Hot reloading is to inject a new files being edited without stop the running app. In this way, you don't lose state of the app which is especially useful when editing UI.
+
+The following video is a video of hotreloading an app running on Tizen TV Emulator.
+
+![test video](docs/test.webp)
+
+
+## How to enable:
+
+
+### Install with Private Feed
+
+Add this feed to Your VS Extension private gallery.
+* https://nui-dali.github.io/Beholder/gallery/feed.xml
+
+
+![extension setup](docs/extension_setup.png)
```

Comment:
Global comment:

We add figures after end of steps/descriptions in the guides. Could you please follow the same and move figures at the end of steps/descriptions?